src/main/java/org/polystat/Polystat.java

```diff
@@ -103,13 +111,24 @@ public final class Polystat implements Callable<Integer> {
 
     /**
      * Main entrance for Java command line.
-     * @param args The args
+     * @param cmdargs The args from the command line.
      */
-    @SuppressWarnings("PMD.DoNotCallSystemExit")
-    public static void main(final String... args) {
-        System.exit(
-            new CommandLine(new Polystat()).execute(args)
-        );
+    @SuppressWarnings({"PMD.DoNotCallSystemExit", "PMD.AvoidCatchingGenericException"})
+    public static void main(final String... cmdargs) {
+        final String[] confargs = new ListOf<String>(
+            new Config(Paths.get(".polystat"))
+        ).toArray(new String[0]);
+        final Polystat config = new Polystat();
+        final Polystat cmdline = new Polystat();
+        new CommandLine(config).parseArgs(confargs);
```

Comment:
@nikololiahim I would do it differently. I would just create a list with args. Add there arguments from `.polystat` file. THEN add there command line args. And then just work with the list, parsing it only once. I'm doing this in Ruby projects: https://github.com/yegor256/texsc/blob/master/bin/texsc#L34-L50

Reply:
I tried doing it like this but the problem was that it would reject it due to arguments being repeated several times. 
For example, if `.polystat` has `--files sandbox` and commandline has `--files src` then it would resolve to `polystat.jar --files sandbox --files src` which is invalid.

Reply:
@yegor256 The solution you described would be similar to using @-files https://picocli.info/#AtFiles

Reply:
@nikololiahim yes, this would be invalid and it would be the problem of a user. But I think we should use a library, which understands that `--files X --files Y` is the same as `--files Y`. But again, it's a problem of a user or the library. But not related to the functionality we implement here.

Reply:
@yegor256 I tried looking for a library solution and stumbled across https://picocli.info/#_default_provider and https://picocli.info/#_propertiesdefaultprovider. The problem with the first is that it delegates the parsing logic (what is an arg and what isn't) to the client so we would have to implement it ourselves. 
The second option could work but it forces the use of the specific format, which from my understanding also isn't desired.

Reply:
@yegor256 In some sense, my proposed solution is provided by a library. We parse 2 sets of args (this is handled by picocli) and then choose which ones we want to keep (this is implemented in the `overrideBy method`). This allows to define a fine-grained overriding behavior, which isn't the best _right now_, but can be improved in the future or refactored to look nicer / leas hacky.

Reply:
@nikololiahim this is what I think is not cool: we _duplicate_ the logic of the library. Would be much simpler (your code will be shorter) if you just put _all_ args into an array and let the lib deal with it. Shorter code of your, and cleaner logic: it's provided by the lib.

Reply:
@yegor256 Ok, I can do that.